data/cjkPts.geojson

```diff
@@ -0,0 +1 @@
哨巩绢"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.69001132062567,43.36003830098169]},"properties":{"name":"螺裹昔轩谬"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.76804370175068,43.37362703079163]},"properties":{"name":"谍龟媳姜瞎"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.77043742133355,43.325828830544545]},"properties":{"name":"冤鸦蓬巷琳"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.66452596219278,43.377082454312294]},"properties":{"name":"栽沾诈斋瞒"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.78052164197288,43.39882235229331]},"properties":{"name":"彪厄咨纺罐"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.74768201610368,43.34347333416988]},"properties":{"name":"桶壤糕颂膨"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.77257029502562,43.353659062164375]},"properties":{"name":"谐垒咕隙辣"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.82229997433387,43.36602478810474]},"properties":{"name":"绑宠嘿兑霉"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.77697418935259,43.39302377957722]},"properties":{"name":"挫稽辐乞纱"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.82932283412902,43.344460090153106]},"properties":{"name":"裙嘻哇绣杖"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.75834764860247,43.43886099129135]},"properties":{"name":"塘衍轴攀膊"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.71397262576284,43.43847474002368]},"properties":{"name":"譬斌祈踢肆"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.78490097957456,43.41708495277058]},"properties":{"name":"坎轿棚泣屡"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.82596603943784,43.37280499147228]},"properties":{"name":"躁邱凰溢椎"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.80376452461223,43.41299359444341]},"properties":{"name":"砸趟帘帆栖"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.73771479115749,43.38964563062126]},"properties":{"name":"窜丸斩堤塌"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.77914081236031,43.36200438697811]},"properties":{"name":"贩厢掀喀乖"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.6885010122187,43.3536150480067]},"properties":{"name":"谜捏阎滨虏"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.6809883295955,43.4279302964151]},"properties":{"name":"匙芦苹卸沼"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.66638088219406,43.356600228775875]},"properties":{"name":"钥株祷剖熙"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.81508020435649,43.44503598067335]},"properties":{"name":"哗劈怯棠胳"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.80562566482058,43.441746093418764]},"properties":{"name":"桩瑰娱娶沫"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.69680728921867,43.37119072206014]},"properties":{"name":"嗓蹲焚淘嫩"}}]}
```

Comment:
Why do we need so many new GeoJSON fixtures for this test? Can we refactor to use a single fixture or rename the fixtures to clearly explain the differences between them?

`bbox.geojson`, `bbox1.geojson`, `bbox2.geojson`, `bbox3.geojson`, `bbox4.geojson`, `data/cjk.geojson`, and `data/cjkPts.geojson`